CEXT-3925: Get all subscribed events & webhook endpoint (#306)
AdobeDocs/commerce-extensibility: src/pages/events/api.md

 '<ADOBE_COMMERCE_URL>/rest/all/V1/eventing/eventSubscribe'
```

## Unsubscribe from events

The `POST /rest/<store_view_code>/V1/eventing/eventUnsubscribe/<event_name>` endpoint unsubscribes from the specified event.

**Header:**

`Authorization: Bearer <administrator token>`

The administrator must be granted access to the `Magento_AdobeCommerceEventsClient::event_unsubscribe` resource.

**Example usage:**

The following cURL command unsubscribes from the `observer.catalog_category_save_after` event.

```bash
curl -i -X POST \
   -H "Authorization:Bearer <AUTH_TOKEN>" \
 '<ADOBE_COMMERCE_URL>/rest/all/V1/eventing/eventUnsubscribe/observer.catalog_category_save_after'
```

## Get a list of all subscribed events

> The `GET /rest/all/V1/eventing/getEventSubscriptions` endpoint returns a list of all subscribed events which are enabled. The response body looks like below:

Reviewer: ```suggestion
The `GET /rest/all/V1/eventing/getEventSubscriptions` endpoint returns a list of all subscribed events that are enabled. The response body is similar to the following:
```

```json
[{
  "name": "observer.catalog_product_save_after.price_check",
  "parent": "observer.catalog_product_save_after",
  "fields": [
    "price",
    "name",
    "sku"
  ],
  "rules": [
    {
      "field": "price",
      "operator": "lessThan",
      "value": "300.00"
    }
  ],
  "destination": "default",
  "priority": false,
  "hipaa_audit_required": false
}]
```

The administrator must be granted access to the `Magento_AdobeCommerceEventsClient::event_subscriptions` resource.

**Example usage:**

> The following cURL command returns returns a list of all subscribed events which are enabled.

Reviewer: ```suggestion
The following cURL command returns returns a list of all subscribed events that are enabled.
```
